examples/plot_post_stack_inversion.py

```diff
@@ -0,0 +1,112 @@
+"""
+Post Stack Inversion - 3D
+=========================
+This illustration demonstrates the implementation of a distributed 3D Post-stack inversion. It involves
+modelling a 3-D synthetic post-stack seismic data from a profile of the subsurface acoustic impedence.
```

Comment:
`3-D` -> `3D` 
`a profile of the subsurface acoustic impedence.` -> `a 3D model of the subsurface acoustic impedence`

Reply:
Done...